News items for 1 Nov (#262)
pathoplexus/pathoplexus: monorepo/website/src/pages/news/2024-11-01-autumn-update.mdx

---
layout: ../../layouts/MdLayout.astro
title: "Pathoplexus Autumn Update"
---

<div>
    <a href="/news">News</a> > <b>Pathoplexus Autumn Update</b>
</div>

> # Two Weeks of Pathoplexus

Reviewer: Months maybe instead of weeks

> # Two Weeks of Pathoplexus

Author: Jesus christ how many times does the title appear, already felt like I'd changed it 5 times 😆

> # Two Weeks of Pathoplexus

Reviewer: `ctrl+f` is your friend 😄

*By the Pathoplexus Team - 1 November 2024*

Despite only being 2 months old, we at Pathoplexus have been excited to already contribute to discussions on data sharing, and start becoming part of the pathogen sharing community. In addition, we've continued to add new features and improvements to Pathoplexus - still largely powered by volunteer effort and donated time!

As the seasons turn, we wanted to share some of our recent news:

### In the media:
- [Lancet Microbe's Elena Dalla Vecchia covered Pathoplexus](https://www.thelancet.com/journals/lanmic/article/PIIS2666-5247(24)00263-5/fulltext), highlighting how Pathoplexus is aiming to improve data sharing through openness, credit, and transparency (13 Sept 2024)

### Visitors and data:

> - Since launch, Pathoplexus had ingested XX new sequences from INSDC-member databases since launch

Reviewer: ```suggestion
- Since launch, Pathoplexus has automatically ingested 53 newly released sequences from INSDC-member databases
```

Reviewer: I'm excluding some 500 CCHF ones that were newly ingested due to a bug in NCBI virus

Reviewer: Also not counted sequence updates, we could have a separate figure for that

Reviewer: We have also got 3 PPX WNV sequences, but might not include this in this update

Author: Ah I think I thought the number was bigger due to the CCHF ones :(

Reviewer: Does your resolving mean you're happy with as is?

Author: Yes I think so, even though it's a bit low number! Still, why not include it 🙃

Author: Sorry if I didn't pull this change specifically - it is the current draft as you proposed it.

> - We are excited to have hit 100 users! (Necessary to upload data but not to view data)

Reviewer 2: ```suggestion
- We are excited to have hit 100 registered users! (Necessary to upload data but not to view data)
```

Or "user accounts"?

Reviewer: I like user accounts

### Executive Board update:
- See recent minutes of [Pathopexus Members meetings, Executive Board meetings, and Executive Board resolutions](https://pathoplexus.org/about/governance/minutes)

### Scientific Advisory Board:
- We’re incredibly pleased to welcome Meera Chand from the UKHSA to the [Scientific Advisory Board](https://pathoplexus.org/about/sab)!

### Pathoplexus Association:
- Pathoplexus is now registered as a tax-exempt organization in Switzerland - any donations made by Swiss residents are tax-deductible

### Software updates:
- You can now use checkboxes to select exactly the sequences you'd like to download from the search/browse pages
- It's now possible to download both the sequence and metadata for individual sequences easily from their 'detail view' page
- Group names and information is now easily editable
- Multi-sequence downloads now have detailed file names, featuring the organism, date, and data type (amino-acid, aligned nucleotide, etc)

> - We've improved how we store author names so that we can more flexibly display them, and pass them in preferred formats on when depositing to INSDC

Reviewer: Maybe overkill, but VS Code has little screencasts showing new features in action, it helps discoverability of the features, makes it easy to know how to use them. We don't need to add them to all issues, only to those where we think this is nice

Author: Sorry I think I don't understand. Would we then add links to these screencasts from the text? or can we put them in the news somehow? I mean that sounds fun if it's not hard, but not fully understanding the suggestion I think

Reviewer: We should be able to add them simply like images (as long as they are gifs, so very low tech, just `!
```suggestion
### Software updates:
- You can now use checkboxes to select exactly the sequences you'd like to download from the search/browse pages
  ![/public/news_oct/feature1.gif]
...
```

Reviewer: If you're open to this, I might have a look at adding one as a trial, but don't block on this

Reviewer: I think I've managed to add one gif now

### Upcoming talks:

>   - Emma Hodcroft at [VGEB](https://escv2024.org/) on Pathoplexus (18-21 Sept, Frankfurt)
>   - Chaoran Chen at [VGEB] (https://asm.org/events/asm-ngs/home) on Loculus (13-16 Oct, Washington DC)

Reviewer 2: ```suggestion
  - Emma Hodcroft at [VGEB](https://coursesandconferences.wellcomeconnectingscience.org/event/virus-genomics-evolution-and-bioinformatics-20241106/) on Pathoplexus (6-8 Nov, Hinxton, UK)
  - Chaoran Chen at [VGEB] (https://coursesandconferences.wellcomeconnectingscience.org/event/virus-genomics-evolution-and-bioinformatics-20241106/) on Loculus (6-8 Nov, Hinxton, UK)
```

---
layout: ../../layouts/MdLayout.astro
title: "Pathoplexus Autumn Update"
---

<div>
    <a href="/news">News</a> > <b>Pathoplexus Autumn Update</b>
</div>

# Pathoplexus Autumn Update

*By the Pathoplexus Team - 1 November 2024*

Despite only being 2 months old, we at Pathoplexus have been excited to already contribute to discussions on data sharing, and start becoming part of the pathogen sharing community (see 'past events' below). In addition, we've continued to add new features and improvements to Pathoplexus - still largely powered by volunteer effort and donated time!

As the seasons turn, we wanted to share some of our recent news:

### In the media:
- [Lancet Microbe's Elena Dalla Vecchia covered Pathoplexus](https://www.thelancet.com/journals/lanmic/article/PIIS2666-5247(24)00263-5/fulltext), highlighting how Pathoplexus is aiming to improve data sharing through openness, credit, and transparency (13 Sept 2024)

### Visitors and data:
- Since launch, Pathoplexus has automatically ingested 53 newly released sequences from INSDC-member databases

> - We are excited to have hit 100 registered users! (Necessary to upload data but not to view data)

Reviewer 2: The parenthesis might be confusing. For example, one might think that 100 registered users are needed to upload data? What about this:

```suggestion
- We are excited to have hit 100 registered users! (Registration is needed to upload data but not to view data.)
```

### Executive Board update:
- See recent minutes of [Pathopexus Members meetings, Executive Board meetings, and Executive Board resolutions](https://pathoplexus.org/about/governance/minutes)

### Scientific Advisory Board:
- We’re incredibly pleased to welcome Meera Chand from the UKHSA to the [Scientific Advisory Board](https://pathoplexus.org/about/sab)!

### Pathoplexus Association:
- The Pathoplexus association has been recognized as tax-exempt in Switzerland. This means any donations made by Swiss residents are now tax-deductible.

### Software updates:
- You can now use checkboxes to select exactly the sequences you'd like to download from the search/browse pages

>   ![GIF showing how to select sequences for download](/public/gifs/news/2024-11-01-autumn-update/checkbox.gif)

Reviewer: ```suggestion
  ![GIF showing how to select sequences for download](/gifs/news/2024-11-01-autumn-update/checkbox.gif)
```

- It's now possible to download both the sequence and metadata for individual sequences easily from their 'detail view' page

> - Group names and information is now easily editable

Reviewer: ```suggestion
- Group names and information are now easily editable
```

- Multi-sequence downloads now have detailed file names, featuring the organism, date, and data type (amino-acid, aligned nucleotide, etc)

> - We've improved how we store author names so that we can more flexibly display them, and pass them in preferred formats on when depositing to INSDC

Reviewer 3: This line sounded a bit odd to me when reading through, should it not be
```suggestion
- We've improved how we store author names so that we can display them more flexibly, and pass them on in preferred formats when depositing to INSDC
```

Reviewer: I think this isn't really a feature, it's a breaking change for INSDC compatibility reasons.

Ok there is a feature and it's that we normalize name now to be sentence case from INSDC, but this is sort of unrelated to the reformatting breaking change

### Users and data:
- Since launch, Pathoplexus has automatically ingested 53 newly released sequences from INSDC-member databases
- We are excited to have hit 100 registered users! (Registration is only needed to upload data and create SeqSets but not to view data.)

### Executive Board update:
- See recent minutes of [Pathopexus Members meetings, Executive Board meetings, and Executive Board resolutions](https://pathoplexus.org/about/governance/minutes)

### Scientific Advisory Board:
- We’re incredibly pleased to welcome Meera Chand from the UKHSA to the [Scientific Advisory Board](https://pathoplexus.org/about/sab)!

### Pathoplexus Association:
- The Pathoplexus association has been recognized as tax-exempt in Switzerland. This means any donations made by Swiss residents are now tax-deductible.

### Software updates:
- You can now use checkboxes to select exactly the sequences you'd like to download from the search/browse pages

> <div class="my-3">
> ![GIF showing how to select sequences for download](/gifs/news/2024-11-01-autumn-update/checkbox.gif)
> </div>

Reviewer: ```suggestion
  ![GIF showing how to select sequences for download](/gifs/news/2024-11-01-autumn-update/checkbox.gif)
```

Div padding no longer necessary due to general css changes
